src/Components/Section.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  FormEvent, useRef, useState, useEffect,
+} from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  onSelect: (value: Todo) => Promise<void>;
+  filteredTodos: Todo[] | null;
```

Comment:
```suggestion
  filteredTodos: Todo[];
```
this prop cannot be null